src/main/asciidoc/reference/elasticsearch-object-mapping.adoc

```diff
@@ -64,8 +64,7 @@ See https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-type
 * `@GeoPoint`: marks a field as _geo_point_ datatype.
 Can be omitted if the field is an instance of the `GeoPoint` class.
 
-NOTE: Properties that derive from `TemporalAccessor` or are of type `java.util.Date` must either have a `@Field` annotation of type `FieldType.Date` and a
-format different from `DateFormat.none` or a custom converter must be registered for this type. +
+NOTE: Properties that derive from `TemporalAccessor` or are of type `java.util.Date` must either have a `@Field` annotation of type `FieldType.Date`.
```

Comment:
I think we should add a new section here about DateFormats and explain how multiple formats are mapped to Elasticsearch and also describe that for a custom format there may be multiple patterns; best would be a sample `@Field` annotation and the mapping that is created in Elasticsearch.

Reply:
I have changed the documentation so that it is clear what values can be set to `format` and `pattern` and I have linked the ES documentation pages.